D/DDSCAT/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "DDSCAT"
+version = v"7.3.3"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://ddscat.wikidot.com/local--files/downloads/ddscat$(version)_220120.tgz", "06f2673a45fcff20b8ed9f37d5bedeb84604d7a9fb10f627096833163769d1e7")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/src
+make ddscat
+mv ddscat "ddscat${exeext}" || true
+install -Dvm 755 "ddscat${exeext}" "${bindir}/ddscat${exeext}"
```

Comment:
I presume
```suggestion
install -Dvm 755 "ddscat" "${bindir}/ddscat${exeext}"
```
would work as well?  Simpler, and without `|| true`

Reply:
No, some Windows platforms failed to produce the correct bin file.

Reply:
For example, https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=20910&view=logs&j=411ac9ab-5c78-57ef-d547-d4acc6737525&t=62223b12-f50e-55d5-058a-3ac90a23eba3&l=361

Reply:
That's not what I said?